ocaml/idl/datamodel_cluster.ml

```diff
@@ -51,8 +51,8 @@ let pool_create = call
     ~doc:"Attempt to create a Cluster from the entire pool"
     ~result:(Ref _cluster, "the new Cluster")
     ~versioned_params:
-            ([{param_type=Ref _network; param_name="network"; param_doc="the single network on which corosync carries out its inter-host communications"; param_release=kolkata_release; param_default=None};
-             {param_type=String; param_name="cluster_stack"; param_doc="simply the string 'corosync'. No other cluster stacks are currently supported"; param_release=kolkata_release; param_default=None};
+          ([{param_type=String; param_name="cluster_stack"; param_doc="simply the string 'corosync'. No other cluster stacks are currently supported"; param_release=kolkata_release; param_default=None};
+            {param_type=Ref _network; param_name="network"; param_doc="the single network on which corosync carries out its inter-host communications"; param_release=kolkata_release; param_default=None};
```

Comment:
minor: I see no functional changes in this hunk, maybe the original order should be kept and then it doesn't show up in the diff.

Reply:
I think these changes are due to originally removing it as part of removing `Cluster.network` uses, then adding it back after deciding to pass a `network` param to `Cluster.pool_create`, so I'll try and change it back